homeassistant/components/bosch_shc/switch.py

```diff
@@ -0,0 +1,288 @@
+"""Platform for switch integration."""
+from boschshcpy import (
```

Comment:
you need to import here `from __future__ import annotations` to fix pylint issue